groups/dps/data.tf

```diff
@@ -0,0 +1,81 @@
+data "aws_ami" "dps" {
+  owners      = [var.ami_owner_id]
+  most_recent = true
+  name_regex  = "^${var.service}-ami-\\d.\\d.\\d"
+
+  filter {
+    name   = "name"
+    values = ["${var.service}-ami-${var.ami_version_pattern}"]
+  }
+}
+
+data "aws_ec2_managed_prefix_list" "shared_services_management" {
+  name = "shared-services-management-cidrs"
+}
+
+data "aws_iam_user" "concourse" {
+  user_name = "concourse-platform"
+}
```

Comment:
This data source doesn't appear to be referenced anywhere. Is it required?

Reply:
Removed in [4268a89](https://github.com/companieshouse/dps-stack/pull/1/commits/4268a89fdf74c76c1e340bd290ab9b60ea02b876).